integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -462,4 +467,106 @@ describe("/admin/product-categories", () => {
       )
     })
   })
+
+  describe("POST /admin/product-categories/:id/products/batch", () => {
+    beforeEach(async () => {
+      await adminSeeder(dbConnection)
+
+      productCategory = await simpleProductCategoryFactory(dbConnection, {
+        id: "test-category",
+        name: "test category",
+      })
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("should add products to a product category", async () => {
+      const api = useApi()
+      const testProduct1 = await simpleProductFactory(dbConnection, {
+        id: "test-product-1",
+        title: "test product 1",
+      })
+
+      const testProduct2 = await simpleProductFactory(dbConnection, {
+        id: "test-product-2",
+        title: "test product 2",
+      })
+
+      const payload = {
+        product_ids: [{ id: testProduct1.id }, { id: testProduct2.id }],
+      }
+
+      const response = await api.post(
```

Comment:
thought: You could pass `expand=products` in order to get the product back with the category right? instead of using the manager below. The idea is that the manager is the last resort wdyt? One question though, if we pass expand products and there is 1000 products in the category, will the 1000 products be returned?

Reply:
The potential for categories returning 100s of products is very likely. `expand=products` for this case should be denied. 

If the consumer would like to get products for a category/categories, they can query the products API with category_ids